pkg/config/config.go

```diff
@@ -70,6 +71,7 @@ type Config struct {
 	AdminPort  int
 	Fs         Fs
 	CouchDB    CouchDB
+	Redis      *redis.Options
```

Comment:
We should have one redis URL per usage: one for cache now, and later one for locks and one for pub/sub